model_compiler/model_converter.py

```diff
@@ -0,0 +1,57 @@
+import requests
+import argparse
+from argparse import ArgumentParser
+
+
+def parse_args():
+    epilog_text = '''
+    Myriad blob compiler in cloud.
+    '''
+    parser = ArgumentParser(epilog=epilog_text,formatter_class=argparse.RawDescriptionHelpFormatter)
+    parser.add_argument("-sh", "--shaves", default=4, type=int,
+                        help="Number of shaves used by NN.")
+    parser.add_argument("-cmx", "--cmx_slices", default=4, type=int,
+                        help="Number of cmx slices used by NN.")
+    parser.add_argument("-NCE", "--NCEs", default=1, type=int,
+                        help="Number of NCEs used by NN.")
+    parser.add_argument("-o", "--output", default=None,
+                        type=str, required=True,
+                        help=".blob output")
+    parser.add_argument("-i", "--input", default=None,
+                        type=str, required=True,
+                        help="model name")
+    options = parser.parse_args()
+    return options
+
+args = vars(parse_args())
+
+PLATFORM="VPU_MYRIAD_2450" if args['NCEs'] == 0 else "VPU_MYRIAD_2480"
+
+url = "http://69.164.214.171:8080/"
```

Comment:
That's luxonis.com, right? Should we change the url to http://luxonis.com:8080 ?